packages/stories-block/src/block/block-types/single-story/embedControls.js

```diff
@@ -151,7 +151,11 @@ const EmbedControls = (props) => {
                       )}
                 </p>
                 {hasPoster && (
-                  <Button onClick={onRemovePoster} isTertiary>
+                  <Button
+                    style={{ margin: '0 12px' }}
+                    onClick={onRemovePoster}
+                    isTertiary
+                  >
```

Comment:
```suggestion
                  <Button onClick={onRemovePoster}>
```

Just removing the `isTertiary` seems to do the trick.

Reply:
See https://wordpress.github.io/gutenberg/?path=/docs/components-button--default

Reply:
Pushed an update -- removed isTertiary + the hard coded margin on the button

Reply:
With the margin removed --- the buttons outline when focused still "bump" into each other when using the keyboard to navigate.

With "VoiceOver" on + keyboard nav
<img width="246" alt="Screen Shot 2022-02-08 at 5 59 03 AM" src="https://user-images.githubusercontent.com/62242/152978875-86920255-c1df-4f6f-8a83-b1c3ce03cc24.png">

Keyboard nav "VoiceOver" off
<img width="267" alt="Screen Shot 2022-02-08 at 6 30 18 AM" src="https://user-images.githubusercontent.com/62242/152978928-e1f0862b-4f65-450d-8d72-c7c516401020.png">

Reply:
In that case, maybe some left margin + removing isTertiary?

Reply:
@spacedmonkey understand your concern about hard coding a margin but I'm not finding other examples in the code base for spacing.

The button group https://wordpress.github.io/gutenberg/?path=/docs/components-buttongroup--default 

Still hard codes a value --- but it applies to both buttons.

```
() => {
	const style = { margin: '0 4px' };
	return (
		<ButtonGroup>
			<Button variant="primary" style={ style }>
				Button 1
			</Button>
			<Button variant="primary" style={ style }>
				Button 2
			</Button>
		</ButtonGroup>
	);
}
```

Reply:
We have a CSS file in our block for adding custom styling if necessary.

Reply:
Re-added the margin here:

https://github.com/GoogleForCreators/web-stories-wp/pull/10510/commits/062fb080cbcfdd2ad9bcbe770e8175486aa7b969

I'm not a fan of hard-coding this either but moving it out to a stylesheet will likely make it harder to track down if / when it needs to be updated.

Open to other suggestions as you see fit.

Reply:
> moving it out to a stylesheet will likely make it harder to track down if / when it needs to be updated.

Fair concern, but now it's an outlier because all our other styles are in a stylesheet.

Also, our stylesheet files are run through RTLCSS.

So you only need to write `margin-left: 12px` and it will be automatically changed to `margin-right: 12px` for RTL.

That can't be done for inline styles.


So I'd just add a className to the Remove button and style that in the CSS file.